src/app/Application.php

```diff
@@ -78,4 +81,12 @@ private static function createContainer($config): Container
 
         return $container;
     }
+
+    public function setupLogging(App $app)
+    {
+        $app->addRoutingMiddleware();
+        $logger = new Logger('error');
+        $logger->pushHandler(new RotatingFileHandler('../../logs/slim-error.log', 30));
```

Comment:
../../ should reference to deployment root :crossed_fingers: 

![axonivy-website-dirs](https://github.com/user-attachments/assets/9a6f0966-db1f-4eb2-92e4-3a22963bf32f)